.github/workflows/ci.yaml

```diff
@@ -27,7 +34,9 @@ jobs:
 
       - name: Install project dependencies
         if: steps.yarn-cache.outputs.cache-hit != 'true' # Over here!
-        run: yarn
+        run: |
+          yarn
+          yarn install
 
       - name: Compile application
         run: yarn build
```

Comment:
I saw this command line failed, the env variable `CI` should be `false` to avoid the warnings,
should I update it to`CI=false yarn run build`?

Reply:
thanks just saw the same. fixed it.